README.md

```diff
@@ -85,6 +95,21 @@ rya.tableprefix=triplestore_  #Rya Table Prefix
 rya.displayqueryplan=true  #To display the query plan
 ```
 
+Here is an example for mongoDB (populate user/userpassword if authentication to mongoDB required):
```

Comment:
the user/password fields are optional, so if there is no auth for mongo, don't include the fields.  empty fields may cause exceptions